test/tests_nso_to_oc/xe/xe_routing_policy.yml

```diff
@@ -0,0 +1,441 @@
+    nso_password: "{{ lookup('env', 'NSO_PASSWORD') }}"
+    device: "{{ lookup('env', 'TEST_DEVICE_XEROUTER') | default('xe1', True) }}"
+  tasks:
+    - block:
+      - name: test set_up
+        tags:
+          - set_up
+        import_role:
+          name: nso-ned-device-configure
+        vars:
+          api_path: mdd:openconfig
+          content: |
+            mdd:openconfig:
+              openconfig-acl:acl:
+                openconfig-acl:acl-sets:
+                  openconfig-acl:acl-set:
+                    - openconfig-acl:name: '111'
+                      openconfig-acl:type: 'ACL_IPV4'
+                      openconfig-acl:config:
+                        openconfig-acl:description: 'extended_test'
+                        openconfig-acl:name: '111'
+                        openconfig-acl:type: 'ACL_IPV4'
+                      openconfig-acl:acl-entries:
+                        openconfig-acl:acl-entry:
+                          - openconfig-acl:sequence-id: 10
+                            openconfig-acl:actions:
+                              openconfig-acl:config:
+                                openconfig-acl:forwarding-action: 'ACCEPT'
+                                openconfig-acl:log-action: 'LOG_NONE'
+                            openconfig-acl:config:
+                              openconfig-acl:description: 'extended_test'
+                              openconfig-acl:sequence-id: 10
+                            openconfig-acl:ipv4:
+                              openconfig-acl:config:
+                                openconfig-acl:destination-address: '0.0.0.0/0'
+                                openconfig-acl:source-address: '0.0.0.0/0'
+                    - openconfig-acl:name: '12'
+                      openconfig-acl:type: 'openconfig-acl-ext:ACL_IPV4_STANDARD'
+                      openconfig-acl:config:
+                        openconfig-acl:name: '12'
+                        openconfig-acl:type: 'openconfig-acl-ext:ACL_IPV4_STANDARD'
+                      openconfig-acl:acl-entries:
+                        openconfig-acl:acl-entry:
+                          - openconfig-acl:sequence-id: 10
+                            openconfig-acl:actions:
+                              openconfig-acl:config:
+                                openconfig-acl:forwarding-action: 'ACCEPT'
+                                openconfig-acl:log-action: 'LOG_NONE'
+                            openconfig-acl:config:
+                              openconfig-acl:sequence-id: 10
+                            openconfig-acl-ext:ipv4:
+                              openconfig-acl-ext:config:
+                                openconfig-acl-ext:source-address: '0.0.0.0/0'
+          api_method: PUT
+
+      - name: NSO configure ip
+        tags:
+          - ip
+        import_role:
+          name: nso-ned-device-configure
+        vars:
+          api_path: config/tailf-ned-cisco-ios:ip
+          content: |
```

Comment:
should take deny statements out of tests, since they should prevent list from being added to OC.